org-code-javabuilder/playground/src/main/java/org/code/playground/PlaygroundSignalKey.java

```diff
@@ -0,0 +1,5 @@
+package org.code.playground;
+
+public enum PlaygroundSignalKey {
+  ADD_IMAGE
```

Comment:
NIT: add a comment to the key (similar to NeighborhoodSignalKey's comments)

Reply:
Also these keys may likely change as we start implementing, so we can remove for now

Reply:
Added a comment that this doesn't actually do anything at the moment, will delete as we actually build :)